plugin/api.go

```diff
@@ -331,6 +331,11 @@ type API interface {
 	// Minimum server version: 5.3
 	GetFileInfo(fileId string) (*model.FileInfo, *model.AppError)
 
+	// GetFile gets content of a file by it's ID
+	//
+	// Minimum Server version: 5.6
```

Comment:
Since we are shipping 5.6 soon, this PR wont make it into 5.6. Can you please change this to 5.8?